src/azure-cli/azure/cli/command_modules/acs/decorator.py

```diff
@@ -436,9 +502,19 @@ def get_ssh_key_value_and_no_ssh_key(self) -> Tuple[str, bool]:
         # these parameters do not need dynamic completion
 
         # validation
-        _validate_ssh_key(
-            no_ssh_key=no_ssh_key, ssh_key_value=ssh_key_value
-        )
+        if not no_ssh_key:
+            try:
+                if not ssh_key_value or not is_valid_ssh_rsa_public_key(
+                    ssh_key_value
+                ):
+                    raise ValueError()
+            except (TypeError, ValueError):
+                shortened_key = truncate_text(ssh_key_value)
```

Comment:
what's the behaviour of truncate_text?
to me even print one shortened_key is not acceptable.

Reply:
Just truncate the string, by default the maximum length is 70 characters. Besides, this is the original behavior defined in `_validate_ssh_key`.